utils/clients/queue_client.py

```diff
@@ -36,6 +31,7 @@ def __init__(self, credentials=None, consumer_name='queue_client'):
         self._connection = None
         self._consumer_name = consumer_name
         self._autoreduce_queues = self.credentials.all_subscriptions
+        self._logger = self._logger.getLogger(__file__)
```

Comment:
Looks like its this line, failing the build

Reply:
Find & replace gone wrong 😬